README.md

```diff
@@ -8,6 +8,11 @@ There are two ways to develop this project:
 - Frontend development only, using the AWS integration environment for everything else. This is the default
 - Full stack local development, using a local dev copy of the API, Keycloak, etc
 
+
+- When moving between full local and frontend only development you should clear your browsers' cookies and local storage as the browser can cache the local Cognito ID, which is incompatible with non-local Cognito. An example of the error you will see if this happens is below:
+
+![Cognito error example](public/images/Cognito-error-example-screenshot.png)
```

Comment:
Could you include the error as text rather than an image? There a couple of reasons for this: if people search for the error text, it would be useful for this to show up as a search result, and putting an image in `public/images` means that it gets deployed with the Play site! 🙃

Reply:
Ah of course!